### PR TITLE
SPUI-155 > Add selection of different API flavors in dev environment

### PR DESCRIPTION
**Pull request by eli1stark**

### CODE REVIEW

File: lib/main_production.dart [hunk 1]
```diff
@@ -31,7 +31,8 @@ Future<void> main() async {
   await Firebase.initializeApp();
   if (!kIsWeb) {
     await PushNotificationService().initialize();
-    await RemoteConfigService().overrideBaseUrls();
+    await RemoteConfigService().fetchAndActivate();
+    RemoteConfigService().overrideApisFrom(0);
```
Comment (VovaMedvediev) [inline lib/main_production.dart:35]: refactor to variable with understandble name or add comment with description of value "0"

File: lib/shared/services/remote_config_service/service/remote_config_service.dart [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import 'dart:convert';
+
+import 'package:firebase_remote_config/firebase_remote_config.dart';
+import '../../../../service/shared/api_urls.dart';
+import '../model/connection_config_model.dart';
+
+class RemoteConfigService {
+  final _config = RemoteConfig.instance;
```
Comment (VovaMedvediev) [inline lib/shared/services/remote_config_service/service/remote_config_service.dart:8]: add comment please with description that RemoteConfigService is our abstraction but RemoteConfig.instance is singletone

File: lib/main_production.dart (hunk 1 above)
Comment (eli1stark) [reply lib/main_production.dart]: done

File: lib/shared/services/remote_config_service/service/remote_config_service.dart (hunk 2 above)
Comment (eli1stark) [reply lib/shared/services/remote_config_service/service/remote_config_service.dart:15]: I made RemoteConfigService a Singleton

**Pull request merged**